shared/oae/bundles/email/default.properties

```diff
@@ -8,4 +11,8 @@ ACTIVITY_EMAIL_SUMMARY_IMMEDIATE_MULTIPLE_ACTORS = New activity is waiting for y
 ACTIVITY_EMAIL_SUMMARY_IMMEDIATE_SINGLE_ACTOR_MULTIPLE_ACTIVITIES = ${actorLink} has been very active recently
 ACTIVITY_EMAIL_SUMMARY_IMMEDIATE_SINGLE_ACTOR_SINGLE_ACTIVITY = ${actorLink} has been active recently
 ACTIVITY_EMAIL_SUMMARY_WEEKLY = Here's what happened in the last week
-
+EMAIL_VERIFY_ADMIN_BODY_EXPLANATION = A request to change the email address of your account has been made by an administrator. If you’d like to use this email address instead, please click the following link.
```

Comment:
Going back over the messages they do still seem awkward and perhaps overly technical. I've set them as this, still retaining the difference between a user changing their own address and an administrator changing the address:

```
EMAIL_VERIFY_ADMIN_BODY_EXPLANATION = We have received a request by an administrator to change the email address of your account. Please click the button to verify your email address. If you were not expecting this change, please ignore this message.
EMAIL_VERIFY_BODY_EXPLANATION = We have received a request to change the email address of your account. Please click the button to verify your email address. If you did not request this change, please ignore this message.
```